doc/whats_new/v0.20.rst

```diff
@@ -181,3 +181,10 @@ Cluster
 - Deprecate ``pooling_func`` unused parameter in
   :class:`cluster.AgglomerativeClustering`. :issue:`9875` by :user:`Kumar Ashutosh
   <thechargedneutron>`.
+
+Changes to estimator checks
+---------------------------
+
+- Allow tests in :func:`estimator_checks.check_estimator` to test functions
+  that accept pairwise data.
+  :issue:`9701` by :user:`Andreas Mueller <amueller>`
```

Comment:
This conventionally mentions the contributor of the fix, not the person who raised the issue.